src/report-changes/storage/couchdb-changes.service.ts

```diff
@@ -117,15 +122,13 @@ export class CouchdbChangesService extends DatabaseChangesService {
           return of(undefined);
         }
 
-        return this.couchdbClient.getDatabaseDocument<EntityDoc>(
-          this.dbUrl,
-          this.databaseName,
-          docId,
-          {
+        return this.couchdbClient.getDatabaseDocument<EntityDoc>({
+          documentId: docId,
+          config: {
```

Comment:
I don't see you providing a customized factory for the ReportChanges module or this service ... Are you really sure this already works without passing the URLs? (if so, remove above class properties dbUrl etc.)